### PR TITLE
Add PR Template

### PR DESCRIPTION
**Pull request by jun-loke**
# General info

**Issue Number**: n/a

**Task description**: 

 - Added PR template to repo

# Testing

**Test coverage:** n/a

### CODE REVIEW

**Review by gdzooks — DISMISSED** [review]
looks good! as we discussed in voicechat 👍

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# General info
+<!--Delete or put N/A for any irrelevant sections-->
+
+**Issue Number**: <!--Issue Number-->
```
Comment (sharon-ho) [inline .github/pull_request_template.md]: Can we have ` number` in lower case to fit with the rest of the headings

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+|<!--** original screenshot **--> | <!--** updated screenshot **-->|
+
+# Testing
+
+**Test coverage:** 
+<!-- Unit tested/manual testing only/ Some actual metrics -->
+
+# Additional notes
+
+**Note to reviewers**
```
Comment (sharon-ho) [inline .github/pull_request_template.md]: Add `:` after `reviewers`

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+<!-- Special instructions for testing this code-->
+
+**To do before merging**
```
Comment (sharon-ho) [inline .github/pull_request_template.md]: Add `:` after `merging`

Comment (jun-loke) [reply .github/pull_request_template.md]: fixed in 7163773

File: .github/pull_request_template.md (hunk 2 above)
Comment (jun-loke) [reply .github/pull_request_template.md]: fixed in 7163773

**Review by gdzooks — COMMENTED** [review]
👍

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+ - Change 2
+ 
+<!-- What changed? What does this do? Provide screenshots if necessary--> 
+| Original   | Updated|
+|:---------------:|:---------------:|
+|<!--** original screenshot **--> | <!--** updated screenshot **-->|
+
+# Testing
+
+**Test coverage:** :
```
Comment (sharon-ho) [inline .github/pull_request_template.md]: Two `:` here, delete the one within the asterisks

**Review by sharon-ho — CHANGES_REQUESTED** [review]

**Review by gdzooks — DISMISSED** [review]

File: .github/pull_request_template.md (hunk 4 above)
Comment (jun-loke) [reply .github/pull_request_template.md]: Fixed in 27db478

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+ - Change 2
+ 
+<!-- What changed? What does this do? Provide screenshots if necessary--> 
+| Original   | Updated|
+|:---------------:|:---------------:|
+|<!--** original screenshot **--> | <!--** updated screenshot **-->|
+
+# Testing
+
+**Test coverage** :
```
Comment (sharon-ho) [inline .github/pull_request_template.md]: There's a space before the `:`

Comment (jun-loke) [reply .github/pull_request_template.md]: Fixed in 8f44b98

**Review by sharon-ho — APPROVED** [review]
Looks good to me!

**Review by gdzooks — APPROVED** [review]
looks good :,3 :+1:

**Pull request merged**